docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +11,16 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-0-1]]
+=== Logstash 6.0.1 Release Notes
+* Elasticsearch Output: We now ignore an event's type field for the purpose of setting the document _type if the cluster is Elasticsearch 6.x or above.
+* Fix bug where plugins that used nested hashes for configuration would not work correctly
+* Improved shutdown process by ensuring that Logstash terminates on second SIGINT. Prior to this release, there were some situations where executing two SIGNINTs would not kill Logstash. (you might want to format SIGINT with back ticks...I can't remember whether we do elsewhere).
+* Fixed a bug where persistent queue could not be opened when in some states
+* Fixed an issue where the aggregate filter would not always shut down correctly
+* Fixed a large performance regression in the grok filter that has been with us since Logstash 5.x . 
```

Comment:
didn't notice before but you have an extra space before the period. Maybe just remove the period because we mostly don't have periods after list items here.